File: code-squad/week02/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        indice =  [i for i in range(len(nums))]
        for pair in combinations(indice, 2):
            if nums[pair[0]] + nums[pair[1]] == target:
                return pair
        return [0, 0]

File: code-squad/week02/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer, Integer> visited = new HashMap<Integer, Integer>();
        for(int i = 0; i< nums.length; i++){
            int a = target - nums[i];
            if(visited.containsKey(a)){
                return new int[]{ visited.get(a), i};
            }
            visited.put(nums[i], i);

        }
        return new int[]{0, 0};
    }
}